erizo_controller/erizoClient/src/webrtc-stacks/BaseStack.js

```diff
@@ -90,7 +94,7 @@ const BaseStack = (specInput) => {
     }
   };
 
-  const setLocalDescForOffer = (isSubscribe, streamId, sessionDescription) => {
+  that.setLocalDescForOffer = (isSubscribe, streamId, sessionDescription) => {
```

Comment:
I'd use another approach rather than making all these methods public. Why do we need to do it specifically?

Reply:
'cause we need to access them in the EdgeStack.js to remove the abs-send-time